packages/flutter_tools/test/general.shard/devfs_test.dart

```diff
@@ -202,7 +201,8 @@ void main() {
         FakeRequest(Uri.parse('http://localhost'), method: HttpMethod.put, responseError: const OSError('Connection Reset by peer')),
         FakeRequest(Uri.parse('http://localhost'), method: HttpMethod.put, responseError: const OSError('Connection Reset by peer')),
         FakeRequest(Uri.parse('http://localhost'), method: HttpMethod.put, responseError: const OSError('Connection Reset by peer')),
-        FakeRequest(Uri.parse('http://localhost'), method: HttpMethod.put)
+        // This is the value of `<int>[1, 2, 3, 4, 5]` run through `osUtils.gzipLevel1Stream`.
```

Comment:
I think this change is better since we're using the real OS utils.

Reply:
uh oh ... is this different on different platforms?

Reply:
I looked over the Dart SDK sources quickly, but didn't see anything platform specific in them.

Reply:
I updated this to encode the expected value on the fly to work around it. But there is either some machine specific or host specific dependencies.

From linux-x64 build bot:

`[31, 139, 8, 0, 0, 0, 0, 0, 4, 3, 99, 100, 98, 102, 97, 5, 0, 244, 153, 11, 71, 5, 0, 0, 0]`

from macos-x64 build bot:

` [31, 139, 8, 0, 0, 0, 0, 0, 4, 19, 99, 100, 98, 102, 97, 5, 0, 244, 153, 11, 71, 5, 0, 0, 0]`